src/main/java/frc/robot/Controls.java

```diff
@@ -421,6 +457,23 @@ private Command getCoralBranchHeightCommand() {
     };
   }
 
+  private Command getSoloCoralBranchHeightCommand() {
```

Comment:
Can we combine getCoralBranchHeightCommand() and getsoloCoralBranchHeightCommand() by passing in the controller you want?